src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -6203,7 +6203,8 @@ void StorageReplicatedMergeTree::alter(
         size_t mutation_path_idx = std::numeric_limits<size_t>::max();
 
         String new_metadata_str = future_metadata_in_zk.toString();
-        ops.emplace_back(zkutil::makeSetRequest(fs::path(zookeeper_path) / "metadata", new_metadata_str, current_metadata->getMetadataVersion()));
+        Int32 metadata_version = tryFixMetadataVersionInZooKeeper();
```

Comment:
Do we need this since we fix the metadata_version when attaching the table?

Reply:
I forgot to remove this code, fixed now